dev/src/dev/deps_graph.clj

```diff
@@ -52,6 +52,14 @@
      :depends-on-namespace (:namespace ns-dep)
      :depends-on-module    (:module ns-dep)}))
 
+(defn external-usages-by-namespace
```

Comment:
Not strictly related but I added this util function to help me find the places using stuff like `metabase.sync.sync-metadata.tables` so I could remove them as allowed external namespaces